core/src/mindustry/ai/types/FormationAI.java

```diff
@@ -97,4 +106,12 @@ public boolean isBeingControlled(Unit player){
     public Vec3 formationPos(){
         return target;
     }
+
+    private static float pathfindCost(Tile tile){
+        return 1f + (tile.floor().isLiquid ? 10f : 0f);
+    }
+
+    private static float navalPathfindCost(Tile tile){
```

Comment:
what's the point of making a separate private method that just returns one? as I see there is only one usage in code.

Reply:
Yes but I am not sure about euristics. It can be modified later. But atm yes - it doesn't do anything.